yt/utilities/on_demand_imports.py

```diff
@@ -299,6 +289,16 @@ def optimize(self):
 
 
 class h5py_imports(OnDemand):
```

Comment:
So I believe that this is correct behavior, but one question I have is if there are any known side-effects of the import of netCDF4.  Does it register anything with h5py (like plugins etc)?

Reply:
To the best of my knowledge, it doesn't